src/adapter/src/coord/sequencer/inner/create_view.rs

```diff
@@ -157,8 +159,8 @@ impl Coordinator {
 
                 let _ = internal_cmd_tx.send(Message::CreateViewStageReady {
                     ctx,
-                    otel_ctx,
                     stage,
+                    span: parent_span,
```

Comment:
Combined with the above comment, instead of passing `parent_span` here, can you pass `_guard.exit()`? Or if we use `.in_scope(...)` you can pass the Span itself.